### ISSUE TITLE
1 click installation

### ISSUE THREAD

**Issue by lexis33** [AkshitIreddy/Interactive-LLM-Powered-NPCs#4]
It would be great to have an option where clicking one button makes everything work. Include Linux, Mac and Windows.

**Comment by AkshitIreddy** [comment]
Yeah, i think i can probably do it for windows later on, not sure if this project will work with linux and mac though.
